IPython/html/static/notebook/less/notificationarea.less

```diff
@@ -30,12 +30,16 @@
     .indicator_area();
 }
 
-.edit_mode .modal_indicator:before {
+.modal_indicator:before {
     .icon(@fa-var-pencil)
 }
 
+.edit_mode .modal_indicator:before {
+    visibility: inherit;
+}
+
 .command_mode .modal_indicator:before {
-    .icon(' ');
+    visibility: hidden;
```

Comment:
This isn't how we should do it. Command mode has no icon, it isn't invisible (I have a non-empty command mode icon, and this would hide it).